README.md

```diff
@@ -102,13 +81,6 @@ High level `make` targets:
 * `system-test`: run the networking/"sanity" tests. Specify
   `CONTIV_NODE_OS=centos` to test on centos instead of ubuntu.
 
-#### Resource Allocation
```

Comment:
@jainvipin 
>  I am thinking we keep the front page to be usage centric

sounds good. May be it could be moved to a more detailed doc like `docs/ConfigDetails.md`.

@erikh 

> Maybe we should have a CONTRIBUTING.md?

There is a contributing.md here `docs/CONTRIBUTING.md`